libraries/botbuilder-core/botbuilder/core/activity_handler.py

```diff
@@ -27,14 +29,15 @@ class ActivityHandler:
             await self.on_message_activity(turn_context)
         elif turn_context.activity.type == ActivityTypes.conversation_update:
             await self.on_conversation_update_activity(turn_context)
+        elif turn_context.activity.type == ActivityTypes.message_reaction:
+            await self.on_message_reaction_activity(turn_context)
         elif turn_context.activity.type == ActivityTypes.event:
             await self.on_event_activity(turn_context)
         else:
             await self.on_unrecognized_activity_type(turn_context)
 
-    async def on_message_activity(  # pylint: disable=unused-argument
-        self, turn_context: TurnContext
-    ):
+    @abstractmethod
```

Comment:
Why is this abstract? wouldn't that mean you _have to_ implement it? I don't think that is the semantic on the other implementations. (If it is, that that seems like a bug)

Reply:
Fixed